### PR TITLE
Added some logic

### PR DESCRIPTION
**Pull request by eliaswolfgang**
Some logic in js

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1 +1,4 @@
 console.log("Hello, World!");
+const hello = "Hello";
+
+hello = "Hello World";
```
Comment (eevargas) [inline script.js:4]: Dude, that is a constant!!!!

**Review by eevargas — CHANGES_REQUESTED** [review]
I left a message . There is a problem.

File: script.js (hunk 1 above)
Comment (eliaswolfgang) [reply script.js:4]: Sorry about that, should be fixed

**Review by eevargas — APPROVED** [review]
Good to go.

**Pull request merged**